### PR TITLE
[Space] Rework navigation

### PR DESCRIPTION
**Pull request by enzozccr**

### CODE REVIEW

File: tests/Controller/EnMarche/Security/AdherentSecurityControllerTest.php [hunk 1]
```diff
@@ -47,7 +47,7 @@ public function testAuthenticationIsSuccessful(): void
 
         $crawler = $this->client->followRedirect();
         $this->assertResponseStatusCode(Response::HTTP_OK, $this->client->getResponse());
-        $this->assertSame(1, $crawler->selectLink('Carl')->count());
+        $this->assertSame('Carl', trim($crawler->filter('span.topbar-dropdown--trigger')->first()->text()));
```
Comment (kevin-verschaeve) [inline tests/Controller/EnMarche/Security/AdherentSecurityControllerTest.php:50]: I think this fails becaus it should be
```suggestion
        $this->assertSame('Carl', trim($crawler->filter('span.js-topbar-dropdown--trigger')->first()->text()));
```
no ?

**Review by ottaviano — APPROVED** [review]

**Review by kevin-verschaeve — APPROVED** [review]

File: templates/components/_dropdown_user_desktop.html.twig [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+                {#                        <div class="b__nudge--top-10">#}
+                {#                            <a href="{{ path('app_funnel_general') }}" class="text--body text--small text--blue--dark link--no-decor">Compléter mon profil</a>#}
+                {#                        </div>#}
+                {#                    </div>#}
+                <ul class="">
+                    {% if is_granted('ROLE_FORMATION_SPACE') %}
+                        <li class="space--5-0"><a href="{{ path('app_formation_home') }}" class="text--body text--small link--no-decor">Formation</a></li>
+                    {% endif %}
+                    <li class="space--5-0"><a href="https://aide.en-marche.fr" rel="noopener noreferrer" target="_blank" class="text--body text--small link--no-decor">Aide</a></li>
+                    {% if is_granted('ROLE_REFERENT') or is_granted('ROLE_COREFERENT') %}
```
Comment (kevin-verschaeve) [inline templates/components/_dropdown_user_desktop.html.twig:37]: can you add a check on the `ROLE_DELEGATED_REFERENT` please ?

Comment (enzozccr) [reply templates/components/_dropdown_user_desktop.html.twig:37]: you mean adding `or is_granted(ROLE_DELEGATED_REFERENT)` ?

Comment (kevin-verschaeve) [reply templates/components/_dropdown_user_desktop.html.twig:37]: yup sorry

**Review by kevin-verschaeve — APPROVED** [review]

**Pull request merged**